### ISSUE TITLE
@UseGuards is not working on BotGateway method

### ISSUE THREAD

**Issue by willjleong** [fjodor-rybakov/discord-nestjs#688]
**Describe the bug**

UseGuards decorator isn't working / guarding when applied to BotGateway class methods.
@discord-nestjs/common === ^3.0.3,
 @discord-nestjs/core" === ^3.0.3,

`bot.gateway.ts`
```typescript
...

@Injectable()
export class BotGateway {
  private readonly logger = new Logger(BotGateway.name);

  constructor(
    @InjectDiscordClient()
    private readonly client: Client,
    private helpService: HelpService,
  ) {}

  @On('messageCreate')
  @UseGuards(MessageFromUserGuard, HelpCommandGuard)
  async onHelpCommand(message: Message): Promise<void> {
    this.logger.log(`Message from bot: ${message.author.bot}`);
    this.helpService.onHelpCommand(message);
  }
 ...
}
```
`message-from-user.guard.ts`
```typescript
import { DiscordGuard } from '@discord-nestjs/core';
import { Message } from 'discord.js';

export class MessageFromUserGuard implements DiscordGuard {
  canActive(event: 'messageCreate', [message]: [Message]): boolean {
    return !message.author.bot;
  }
}
```

`help-command.guard.ts`
```typescript
import { DiscordGuard } from '@discord-nestjs/core';
import { Message } from 'discord.js';

export class HelpCommandGuard implements DiscordGuard {
  canActive(event: 'messageCreate', [message]: [Message]): boolean {
    return message.content.split(' ')[0] === '!help';
  }
}

```

**To Reproduce**

1. When testing,`onHelpCommand` is executing even though I have a guard defined just like in the examples and I am receiving messages from the bot, creating an infinite loop since `!help` is in the `!help` message response from the bot. 

**Expected behavior**
onHelpCommand should only enter the function if Guards pass, in this case if the message comes from a user and the text starts with `!help`. I like how the Guard Decorator is quite clean so I'd like to use it but for some reason it is not working.

**Comment by fjodor-rybakov** [comment]
I was unable to reproduce your issue. Most likely you are [importing](https://github.com/fjodor-rybakov/discord-nestjs/tree/master/packages/core#:~:text=with%20%40UseGuards%20decorator-,%E2%9A%A0%EF%B8%8F,Import%20%40UseGuards%20from%20the%20%40discord%2Dnestjs/core%20package,-/*%20bot.gateway.ts) `@UseGuards` not from `@discord-nestjs/core`

**Comment by fjodor-rybakov** [comment]
Fixed in 3.0.4 version
